### ISSUE TITLE
Create Accordion Control Without Panel Markup

### ISSUE THREAD

**Issue by ascendantofrain** [twbs/bootstrap#16298]
Is there any feasible way to create a bootstrap accordion control without having to use the 'panels' markup?? I have the collapse control working but I would like to be able to target parents for each section in the control.

In other words, I would like the "Revenue" and "Margin" buckets to toggle each other and inside of "Revenue" have "Revenue Current", "Revenue Base", and "Revenue Current vs. Base" toggle each other without disturbing the parent layers.

http://jsfiddle.net/ascendantofrain/e2t3xu81/2/

**Comment by twbs-lmvtfy** [comment]
Hi @ascendantofrain!

You appear to have posted a live example (http://fiddle.jshell.net/Lp903gc5/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 46, column 17 thru line 46, column 127: Attribute ```` href ```` not allowed on element ```` span ```` at this point.
* line 46, column 17 thru line 46, column 127: Element ```` span ```` is missing one or more of the following attributes: ```` role ````.
* line 52, column 25 thru line 52, column 75: Attribute ```` href ```` not allowed on element ```` span ```` at this point.
* line 79, column 25 thru line 79, column 75: Attribute ```` href ```` not allowed on element ```` span ```` at this point.
* line 106, column 25 thru line 106, column 76: Attribute ```` href ```` not allowed on element ```` span ```` at this point.
* line 156, column 17 thru line 156, column 128: Attribute ```` href ```` not allowed on element ```` span ```` at this point.
* line 156, column 17 thru line 156, column 128: Element ```` span ```` is missing one or more of the following attributes: ```` role ````.
* line 162, column 25 thru line 162, column 76: Attribute ```` href ```` not allowed on element ```` span ```` at this point.
* line 189, column 25 thru line 189, column 76: Attribute ```` href ```` not allowed on element ```` span ```` at this point.
* line 216, column 25 thru line 216, column 76: Attribute ```` href ```` not allowed on element ```` span ```` at this point.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Duplicate of #10966.

> Is there any feasible way to create a bootstrap accordion control without having to use the 'panels' markup??

Not currently. We hope to fix that in Bootstrap v4 though.
